Habitica/src/main/java/com/habitrpg/android/habitica/ui/views/CurrencyView.kt

```diff
@@ -29,12 +29,15 @@ class CurrencyView : android.support.v7.widget.AppCompatTextView {
                 attrs,
                 R.styleable.CurrencyViews,
                 0, 0)
-        lightBackground = attributes?.getBoolean(R.styleable.CurrencyView_hasLightBackground, true) ?: true
+        lightBackground = attributes?.getBoolean(R.styleable.CurrencyView_hasLightBackground,
+                true) ?: true
+        visibility = GONE;
```

Comment:
I set visibility to GONE when the view is first created so that the UI does not show hourglass, gem, and coin icons before these values are loaded. I'm not very familiar with Kotlin, so if anyone knows of a way to do this without having to add it to both constructors, I'd really appreciate it.